pkg/distributor/otel_test.go

```diff
@@ -245,9 +245,9 @@ func TestHandlerOTLPPush(t *testing.T) {
 		series   []prompb.TimeSeries
 		metadata []mimirpb.MetricMetadata
 
-		compression bool
-		encoding    string
-		maxMsgSize  int
+		compression            bool
+		encoding               string
+		maxDecompressedMsgSize int
```

Comment:
Unnecessary change.

Reply:
Agree w/ @pstibrany.

```suggestion
		maxMsgSize int
```

Reply:
renaming is done in commit https://github.com/grafana/mimir/commit/1ba1bcb4854d016dfada97077fc1696ea23c84e3